### PR TITLE
Django Upgrade Release Notes

### PR DESCRIPTION
**Pull request by srpearce**
### Date Needed

This PR needs to be merged when the Django 1.8 upgrade is complete. 
### Reviewers
- [ ] Tech review: @macdiesel or @doctoryes 
- [ ] Doc team review (copy edit): @catong or @lamagnifica  
### HTML Version

http://draft-1117-django-release-notes.readthedocs.org/en/latest/2015/11-17-2015.html
### Post-review
- [ ] Squash commits

### CODE REVIEW

**Comment by macdiesel** [comment]
:+1:

File: en_us/release_notes/source/2015/11-17-2015.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+###################################
+17 November 2015
+###################################
+
+The following information reflects what is new in the edX Platform this week.
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/11-17-2015.rst:6]: maybe repeat the same sentence from openedx_1117_2015.rst, lines 2-3, here as a note? would alert readers more immediately that they might not actually be interested in reading the notes...

File: en_us/release_notes/source/2015/openedx/openedx_1117_2015.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
+
+EdX is not issuing a platform release this week. Instead, edX has completed an
+upgrade from Django 1.4 to Django 1.8.
+
+===================
+Django 1.8 Upgrade
+===================
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/openedx/openedx_1117_2015.rst:7]: maybe an orienting sentence like, EdX uses the Django web framework to develop its web applications: Studio, the LMS, and Insights.
(I checked with Brian and he's ok with this)

File: en_us/release_notes/source/2015/openedx/openedx_1117_2015.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  initial "0001" Django schema migration.
+
+* Data migrations that seed data are now idempotent. All other data migrations
+  have been discarded.
+
+Model Meta Classes
+*********************
+
+* The ``app_label`` attribute has been added to several different models. For more information, see `app_label`_ on the `Django website`_.
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/openedx/openedx_1117_2015.rst:61]: wrap-a-roni

**Comment by doctoryes** [comment]
:+1:

File: en_us/release_notes/source/2015/openedx/openedx_1117_2015.rst [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+Model Migrations
+*********************
+
+* All Django Python South schema migrations have been incorporated into each
+  initial "0001" Django schema migration.
+
+* Data migrations that seed data are now idempotent. All other data migrations
+  have been discarded.
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/openedx/openedx_1117_2015.rst:54]: ok, I had to look that up.

File: en_us/release_notes/source/2015/openedx/openedx_1117_2015.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/release_notes/source/2015/openedx/openedx_1117_2015.rst:7]: also, maybe a statement along the lines of, This upgrade does not result in any user-visible changes.

**Comment by lamagnifica** [comment]
a coupla suggestions, :+1:

**Pull request merged**